### ISSUE TITLE
Package from PyPi doesn't not work

### ISSUE THREAD

**Issue by goldyfruit** [OpenVoiceOS/ovos-ww-plugin-precise-lite#3]
When `ovos-ww-plugin-precise-lite` is installed via PyPi, the `ovos-listener` return this error _(as if the library was not installed)_.

```
2023-03-25 01:45:34.715 - voice - ovos_listener:create_hotword_engines:337 - INFO - creating hotword engines
2023-03-25 01:45:34.716 - voice - ovos_plugin_manager.wakewords:load_module:85 - INFO - Loading "hey_mycroft" wake word via ovos-ww-plugin-precise-lite
2023-03-25 01:45:34.851 - voice - ovos_plugin_manager.utils:load_plugin:118 - WARNING - Could not find the plugin PluginTypes.WAKEWORD.ovos-ww-plugin-precise-lite
2023-03-25 01:45:34.851 - voice - ovos_plugin_manager.wakewords:create_hotword:105 - ERROR - Failed to load hotword: hey_mycroft - ovos-ww-plugin-precise-lite
2023-03-25 01:45:34.852 - voice - ovos_plugin_manager.wakewords:create_hotword:106 - ERROR - Wake Word plugin ovos-ww-plugin-precise-lite not found
Traceback (most recent call last):
  File "/home/ovos/.venv/lib/python3.11/site-packages/ovos_plugin_manager/wakewords.py", line 103, in create_hotword
    return cls.load_module(module, hotword, ww_config, lang, loop)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ovos/.venv/lib/python3.11/site-packages/ovos_plugin_manager/wakewords.py", line 91, in load_module
    raise ValueError(f'Wake Word plugin {module} not found')
ValueError: Wake Word plugin ovos-ww-plugin-precise-lite not found
```

When `ovos-ww-plugin-precise-lite` is installed via GitHub, the listener starts without any issue and the wake word is loaded.
```
2023-03-25 01:47:39.572 - voice - ovos_listener:create_hotword_engines:337 - INFO - creating hotword engines
2023-03-25 01:47:39.573 - voice - ovos_plugin_manager.wakewords:load_module:85 - INFO - Loading "hey_mycroft" wake word via ovos-ww-plugin-precise-lite
2023-03-25 01:47:39.722 - voice - ovos_plugin_manager.wakewords:load_module:92 - INFO - Loaded the Wake Word plugin ovos-ww-plugin-precise-lite
INFO: Created TensorFlow Lite XNNPACK delegate for CPU.
```

**Comment by ChanceNCounter** [comment]
Can repro. Can no longer *not* repro. Wtf? Have not investigated.

**Comment by goldyfruit** [comment]
> Can repro. Can no longer _not_ repro. Wtf? Have not investigated.

I'm using Python 3.11.2, maybe it has not been tested with this version.

**Comment by JarbasAl** [comment]
0.1.1 seems to have this issue, fixed in 0.1.2
